sklearn/impute/tests/test_impute.py

```diff
@@ -971,6 +971,58 @@ def test_iterative_imputer_catch_warning():
     assert not np.any(np.isnan(X_fill))
 
 
+@pytest.mark.parametrize("min_value, max_value, correct_output",
+                         [(0,  # scalars cast to the correct arrays?
+                           100,
+                           np.array([[0] * 3, [100] * 3])),
+                          ([0, 0, 0],  # vectors cast to the same
+                           [100, 100, 100],  # correct arrays?
+                           np.array([[0] * 3, [100] * 3])),
+                          (None,  # None cast to vector of inf?
+                           None,
+                           np.array([[-np.inf] * 3, [np.inf] * 3])),
+                          (-np.inf,  # inf cast to vector of inf?
+                           np.inf,
+                           np.array([[-np.inf] * 3, [np.inf] * 3])),
+                          ([-5, 5, 10],  # vectors processed properly?
+                           [100, 200, 300],
+                           np.array([[-5, 5, 10], [100, 200, 300]])),
+                          ([-5, None, 10],  # vectors of numbers+None
```

Comment:
We should not support the case that we have `None` when this is an array.

Reply:
Done!

Reply:
Done!